src/bandersnatch/utils.py

```diff
@@ -88,17 +87,19 @@ def update_safe(filename: str, **kw: Any) -> Generator[IO, None, None]:
 
     """
     with tempfile.NamedTemporaryFile(
-            dir=os.path.dirname(filename), delete=False,
-            prefix=os.path.basename(filename) + '.', **kw) as tf:
+        dir=os.path.dirname(filename),
+        delete=False,
+        prefix=os.path.basename(filename) + ".",
```

Comment:
NIT: could f"" - No big deal.

Reply:
I'll follow up with https://github.com/asottile/pyupgrade for some free f-string goodness :D